index.html

```diff
@@ -154,21 +154,6 @@ <h3>Closure</h3>
       <section id="Team" class="featured_section">
         <h2>featured Speakers</h2>
         <ul class="teams"></ul>
-        <div class="see-more">
-          <button type="submit">
-            <span>MORE</span> <i class="fa fa-chevron-down"></i>
-          </button>
-        </div>
-      </section>
-      <section id="Partner" class="partners_section">
-        <h2 class="partners-title">Our Partners</h2>
-        <ul class="partner-list d-flex flex-center">
-          <li><img src="./images/partners/1.png" alt="partner_1" /></li>
-          <li><img src="./images/partners/2.png" alt="partner_2" /></li>
-          <li><img src="./images/partners/3.png" alt="partner_3" /></li>
-          <li><img src="./images/partners/4.png" alt="partner_4" /></li>
-          <li><img src="./images/partners/5.png" alt="partner_5" /></li>
-        </ul>
       </section>
     </main>
     <footer id="footer" class="footer_container d-flex">
```

Comment:
- [ ] (OPTIONAL) The home page has to have a dark partner's section and a white footer while the about page only has a dark footer on the desktop version. Kindly adjust your home page as it was alright before by adding the dark partner's section with a white footer 👍 
- Your design:
![Annotation 2022-04-22 040411](https://user-images.githubusercontent.com/85113567/164588704-b95b6697-373d-43c3-9870-22f898d59087.jpg)
- Required design:
![Annotation 2022-04-22 040358](https://user-images.githubusercontent.com/85113567/164588725-e936ece9-8653-49e1-910d-9ce68d57149c.jpg)

- [ ] (OPTIONAL) In the mobile version, the about page has a dark partner's section and a white footer. It would be great if you could implement this.
- Your design:
![Annotation 2022-04-22 040132](https://user-images.githubusercontent.com/85113567/164589197-d6797d1b-19c2-496a-9b7f-23cb548fbe5f.jpg)
- Required design:
![Annotation 2022-04-22 040208](https://user-images.githubusercontent.com/85113567/164589225-bbf4654e-bd06-4b2f-9641-e2152152a772.jpg)



- [ ] (OPTIONAL) Also, it would be great if you could centralize the text and logo on the footer and reduce the spacing between them on both pages so as to follow the design closely 💯 
- Your design:
![Annotation 2022-04-22 040309](https://user-images.githubusercontent.com/85113567/164588937-a34af75b-8b3b-4c9b-babb-fb1320cd1103.jpg)
- Required design:

![Annotation 2022-04-22 040320](https://user-images.githubusercontent.com/85113567/164588978-aaaf12d2-d177-4294-8b3c-43cc2c1b0c81.jpg)